android/src/main/java/com/voximplant/foregroundservice/Constants.java

```diff
@@ -14,4 +14,5 @@
     static final String ERROR_SERVICE_ERROR = "ERROR_SERVICE_ERROR";
     static final String ERROR_ANDROID_VERSION = "ERROR_ANDROID_VERSION";
 
+    static final String BUTTON_PRESSED = "BUTTON_PRESSED";
```

Comment:
Please change the constant value to some package specific to avoid potential collisions with other react-native packages